docs/providers/documentation/gitlab-provider.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "GitLab Provider"
+sidebarTitle: "GitLab Provider"
+description: "GitLab provider is a provider used to query data and creating issues in GitLab"
+---
+
+## Inputs
+
+The `query` function take following parameters as inputs:
+
+- `host` (required): GitLab host name of the project.
+- `PAT` (required): Your Personal Access Token with proper scopes.
+
+The `notify` function take following parameters as inputs:
+
+- `id` (required): The global ID or path of the project.
+- `title` (required): Title of the Issue/Ticket.
+- `description` (optional): Description for the Issue.
+- `labels` (optional): Issue labels seperated by a Comma.
+- `issue_type` (optional): Issue type name. One of `issue`, `incident`, `test_case` or `task`. Default is `issue`.
+
+## Outputs
+
+## Authentication Parameters
+
+The `notify` function requires an `personal_access_token` from GtLab.
```

Comment:
type in GtLab

also please add basic instruction how to get these
also - authentication is not part of notify